jupyter-book/deconvolution/deconv_covid_MuSiC_v0.ipynb

```diff
@@ -0,0 +1,2416 @@
+{
```

Comment:
<p>Can we add a sentence what these files contain and how they were generated? </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/cross-modal-single-cell-best-practices/pull/93/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>More than what we wrote in the beginning of the section: "The bulk data contains 49 bulk whole blood RNA-seq samples collected from 39 COVID-19 patients and 10 healthy controls"?</p>

Reply:
<p>Okay now?</p>

Reply:
<p>Don't see any new changes here? Think we need explanations on where the "BulkSmall.txt" and "annoSmall.txt" files are coming from (experimental metadata? Generated by some tool?) and what they contain. State explicitly what they contain or print the head of those files, please.</p>

Reply:
<p>I gave an explanation and a reference. I'm not sure what else is needed. Metadata is metadata, it can contain many things from different sources. I explained what the RNA-seq matrix contain and it source. </p>

Reply:
<p>Yeah, but nobody sees what's inside these files. Please show the head of them, please. Newbies don't know what these files look like.</p>